.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -13,7 +13,6 @@ assignees: ''
 
 ---
 **Contributor Resources**
-- [Meshery documentation site](https://meshery.layer5.io/docs/)
-- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
-- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
-
+- [Service Mesh Performance documentation site](https://smp-spec.io/docs)
```

Comment:
https://smp-spec.io/docs is showing page not found.

Reply:
@Vineet-Sharma29, good catch. We don't have this page, yet. In the meantime, we can use https://github.com/layer5io/service-mesh-performance/tree/master/docs

Reply:
Never mind. I see now that there is a subsequent line that points to this README already. We can delete line 16 altogether for now.